javascript/core/flow-control/data-type-comparison-in-switch-statements.md

```diff
@@ -82,8 +82,8 @@ switch (???) {
 
 * `x`
 * `case`
-* `1`
 * `'1'`
+* `1`
```

Comment:
Actually, the original was correct, we should use `1` (number) instead of `'1'` (string).

Can be tested with this simple example:

```js
var x = 1;
switch (x) {
  case '1':
    console.log('this gets skipped');
  case 1:
    console.log('this runs');
}
// code above logs "this runs"
```